tests/integration/api/v1/recovery/post.test.js

```diff
@@ -129,21 +131,21 @@ describe('POST /api/v1/recovery', () => {
 
       const responseBody = await response.json();
 
-      expect(response.status).toEqual(400);
-
+      expect(response.status).toBe(201);
```

Comment:
Outra melhoria que venho fazendo nos testes que crio ou altero é adicionar `.soft` nos `expect(response.status)`, pois assim, em caso de erro, podemos ver também qual é a diferença em `expect(responseBody)`:

```suggestion
      expect.soft(response.status).toBe(201);
```